### ISSUE TITLE
train.py recent changes

### ISSUE THREAD

**Issue by scamianbas** [ultralytics/yolov5#522]
## ❔Question
Why some tensorboard data metrics disappeared ?

## Additional context
Hello, 

is it possible to put back _**'metrics/F1'**_ (line 379) ?
F1-score is a very useful metric?

And also **_tb_writer = SummaryWriter(comment=opt.name)_** (line 487) instead of _**tb_writer = SummaryWriter(log_dir=increment_dir('runs/exp', opt.name))**_  ?
A log name built with date, time and hostname is much more useful than this **_exp_** thing ...

IMHO

Thanks !

**Comment by glenn-jocher** [comment]
exp is the recommended prefix in the tensorboard documentation for experiment tracking.

F1 was incorrectly labeled as such before, this bug has been fixed and tensorboard is currently displaying correct labels.

**Comment by scamianbas** [comment]
I had no problems with F1 before and it looks like F1 is not in tags list anymore ?
```
tags = ['train/giou_loss', 'train/obj_loss', 'train/cls_loss',
                        'metrics/precision', 'metrics/recall', 'metrics/mAP_0.5', 'metrics/mAP_0.5:0.95',
                        'val/giou_loss', 'val/obj_loss', 'val/cls_loss']
```

![image](https://user-images.githubusercontent.com/3215795/88487782-5e21d980-cf88-11ea-94bf-2345bcb4262d.png)

Would it be possible to have a command line to choose between exp and the previous log naming ?

Thank you very much.

**Comment by glenn-jocher** [comment]
@scamianbas F1 was incorrectly labelled as such before. You were seeing mAP@0.5:0.95 then and now, the label has simply been corrected. F1 was never displayed. In any case, F1, P and R are all relative metrics that will vary greatly depending on what the user selected confidence they are calculated at, they are not absolute metrics the way mAP is. In laymans terms P, R and F1 vary depending on where on the P-R curve you choose to look. mAP is the area under the entire curve.

The exact naming is subjective of course, I think most people would prefer the simplicity of the new design, especially when running commands for saved weights like python test.py --weights runs/exp0/weights/last.pt. If we hear a lot of feedback otherwise we can revert the design, but so far we haven't.
